### PR TITLE
Microsoft.windows sdk.10.0.20348 10.0.20348.0 0f8bc53c dd8e 4fff b25d b248f609cf74

### PR DESCRIPTION
**Pull request by BrandonWanHuanSheng**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/124858)

### CODE REVIEW

File: manifests/m/Microsoft/WindowsSDK/10/0/20348/10.0.20348.0/Microsoft.WindowsSDK.10.0.20348.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: Microsoft.WindowsSDK.10.0.20348
+PackageVersion: 10.0.20348.0
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://www.microsoft.com/
+PrivacyUrl: https://docs.microsoft.com/en-us/windows/privacy/
+PackageName: Windows Software Development Kit
+PackageUrl: https://developer.microsoft.com/windows/downloads/windows-sdk/
+License: Copyright (c) Microsoft Corporation. All rights reserved.
+ShortDescription: The Windows SDK (10.0.20348.*) for Windows 10 provides the latest headers, libraries, metadata, and tools for building Windows apps.
```
Comment (BrandonWanHuanSheng) [inline manifests/m/Microsoft/WindowsSDK/10/0/20348/10.0.20348.0/Microsoft.WindowsSDK.10.0.20348.locale.en-US.yaml:13]: Only this one having some of the problem.

**Comment by BrandonWanHuanSheng** [comment]
I also see that one could be a merge path. It might be here.

**Comment by BrandonWanHuanSheng** [comment]
It seems this pipelines is taking longer than expected.

File: manifests/m/Microsoft/WindowsSDK/10/0/20348/10.0.20348.0/Microsoft.WindowsSDK.10.0.20348.installer.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Created using wingetcreate 1.5.5.0
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
 
-PackageIdentifier: Microsoft.WindowsSDK.10.0.2104
+PackageIdentifier: Microsoft.WindowsSDK.10.0.20348
 PackageVersion: 10.0.20348.0
```
Comment (ArchGuyWu) [inline manifests/m/Microsoft/WindowsSDK/10/0/20348/10.0.20348.0/Microsoft.WindowsSDK.10.0.20348.installer.yaml:5]: PackageVersion should be 10.1.20348.1

File: manifests/m/Microsoft/WindowsSDK/10/0/20348/10.0.20348.0/Microsoft.WindowsSDK.10.0.20348.locale.en-US.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Created using wingetcreate 1.5.5.0
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
 
-PackageIdentifier: Microsoft.WindowsSDK.10.0.2104
+PackageIdentifier: Microsoft.WindowsSDK.10.0.20348
 PackageVersion: 10.0.20348.0
```
Comment (ArchGuyWu) [inline manifests/m/Microsoft/WindowsSDK/10/0/20348/10.0.20348.0/Microsoft.WindowsSDK.10.0.20348.locale.en-US.yaml:5]: PackageVersion should be 10.1.20348.1

File: manifests/m/Microsoft/WindowsSDK/10/0/20348/10.0.20348.0/Microsoft.WindowsSDK.10.0.20348.yaml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Created using wingetcreate 1.5.5.0
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
 
-PackageIdentifier: Microsoft.WindowsSDK.10.0.2104
+PackageIdentifier: Microsoft.WindowsSDK.10.0.20348
 PackageVersion: 10.0.20348.0
```
Comment (ArchGuyWu) [inline manifests/m/Microsoft/WindowsSDK/10/0/20348/10.0.20348.0/Microsoft.WindowsSDK.10.0.20348.yaml:5]: PackageVersion should be 10.1.20348.1

**Review by ArchGuyWu — APPROVED** [review]

**Comment by BrandonWanHuanSheng** [comment]
But it is not, It write display version entries as I see. Suppose need to matches with 0 file

**Comment by BrandonWanHuanSheng** [comment]
I failed to start the excel.

**Comment by BrandonWanHuanSheng** [comment]
My one of the PR keep mentioned stephengillie

**Comment by BrandonWanHuanSheng** [comment]
https://dev.azure.com/ms/ed6a5dfa-6e7f-413b-842c-8305dd9e89e6/_apis/build/builds/510914/logs/18

**Comment by BrandonWanHuanSheng** [comment]
I renamed the branches.

**Pull request closed without merging**